README.md

```diff
@@ -2,57 +2,82 @@
 
 A Kubernetes controller that provides discovery service for Istio mesh federation.
```

Comment:
As this is the first sentence people will see for the project, I think it could be expanded a bit to introduce the project and describe the controller in more in depth. Perhaps something like (borrowed from your other document):

"This project implements Istio mesh federation using a Kubernetes controller that provides an API for managing multi-mesh communication, implements service discovery and automates the management of Istio configuration."

Reply:
Thanks, done.